meeting-minutes/2024-10-16.md

```diff
@@ -0,0 +1,147 @@
+| Omar       | Santos     | Cisco Systems                                               | Co-Chair                  | YES     |
+| Pablo      | Quiroga    | Qualys, Inc.                                                | Voting Member             | NO      |
+| Peter      | Gephardt   | IBM                                                         | Member                    | NO      |
+| Przemyslaw | Roguski    | Red Hat                                                     | Voting Member             | YES     |
+| Shridhar   | Chari      | Cisco Systems                                               | Member                    | YES     |
+| Sonny      | van Lingen | Huawei Technologies Co., Ltd.                               | Voting Member             | YES     |
+| Stefan     | Arntzen    | Huawei Technologies Co., Ltd.                               | Voting Member             | YES     |
+| Stefan     | Hagen      | Individual                                                  | Secretary, taking notes   | YES     |
+| Thomas     | Proell     | Siemens                                                     | Member                    | NO      |
+| Thomas     | Schaffer   | Cisco Systems                                               | Voting Member             | YES     |
+| Thomas     | Schmidt    | Federal Office for Information Security (BSI) Germany       | Voting Member             | YES     |
+| Tobias     | Limmer     | Siemens                                                     | Member                    | NO      |
+
+Quorum was reached (11 voting members present)
+
+## 1.3 Procedures for this meeting (Moderator)
+
+## 1.4 Approval of agenda
+
+* Roll Call  
+* Updates:  
+  * Discussion on various term use across the industry  
+* Approval of previous meeting minutes (motions carried out per e-mail motions)  
+* Review of outstanding issues and pull requests marked for TC discussion: https://github.com/oasis-tcs/openeox  
+* Next steps  
+
+## 1.5 Approval of previous minutes (Moderator)
+
+None (motions carried out already per e-mail motions).
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+None
+
+## 1.7 Identification of TC voting members (Secretary)
+
+### 1.7.1 Prospective voting members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Secretary)
+
+- Scheduled Teleconferences (Wednesday at 09:00 PDT / 12:00 EDT / 18:00 CEST / **16:00** UTC for 1 hour)
+
+    ```
+    November 29, 2024 (normal time switch back done, same time on wall clock)
```

Comment:
I checked the TC's calendar and it lists `November 20, 2024` as next meeting. Did that change?

Reply:
We meet on Wednesdays and 2024-11-29 is a Friday. So maybe it is a typo. @santosomar can you please kindly confirm?

Reply:
It's a typo. I'm also working with the OASIS staff to update the calendar in the TC workspace. I'm expecting to have all the new invites ready early next week.